journals/apps/journals/blocks.py

```diff
@@ -90,3 +100,12 @@ class JournalImageChooserBlock(ImageChooserBlock):
     '''JournalImageChooserBlock component'''
     def get_searchable_content(self, value):
         return ['Image: ' + value.title]
+
+    def get_api_representation(self, value, context=None):
+        return {
+            'title': value.title,
+            'width': value.width,
+            'height': value.height,
+            'image_id': value.id,
+            'url': value.file.url
```

Comment:
I would see if `value.file.url` is the same as getting the original rendition (done via `value.get_rendition('original')`). I *think* `file.url` uses an internal url and the renditions are where we are supposed to expose from. 

Or I could be totally wrong. Just something to possibly look into.

Reply:
Here is what I found. When you first upload an image into wagtail it is stored in:
`/media/original_images/<filename>`
Then using `value.file.url` will point to that url.
When you call `image.get_rendition('<size>')` that causes a copy of the file to be made with the given rendition size spec in `/media/images/<filename>.<size>` and `value.get_rendition('original').file.url` will point here. Weird thing is the real original in `/media/original_images/<filename>` seems to be larger than the file stored after running `get_rendition('original')` and not sure why.

It seems this is useful when you want to store scaled versions (like thumbnails, etc) so you don't need to manipulate each time. See http://docs.wagtail.io/en/v2.0/advanced_topics/images/renditions.html

So I don't think we want to use the rendition, unless we want to do scaling on the server? But not entirely sure